### DISCUSSION TITLE
Disabling Qtile bar

### DISCUSSION THREAD

**Discussion by antidoid** [qtile/qtile#3666] [Q&A] [answered]
Hey I'm using polybar and the systray doesn't work because the default qtile bar is handling systray. Do you have any idea how to kill the qtile bar. I'm able to hide it but that doesn't disable the qtile bar hence it handles the systray which I don't want it to handle.

 Polybar log says: warn: Systray selection already managed (window=0x0400125)
When running the command: xprop -id 0x0400125
I get: 
            _NET_SYSTEM_TRAY_VISUAL(VISUALID): visual id # 0x21
            WM_STATE(WM_STATE):
                                                  window state: Normal
                                                  icon window: 0x0

I'm new in qtile so I have no idea how to handle the bar.

**Comment by elParaguayo** [comment, marked as answer]
Remove the `Systray` widget from your bar. No point having it if you're not using it.

Or just remove the bar rather than hiding it.

**Reply by antidoid** [reply]
I have removed the bar from my config, but it still ends up showing.

**Reply by antidoid** [reply]
Is there a way to completely remove the bar?

**Reply by antidoid** [reply]
Here's my [config](https://pastebin.com/vMrm2WJg)

**Reply by elParaguayo** [reply]
You don't have a `screens` variable at all (you removed the screen, not the bar). This means it will be replaced by the default value which includes a bar with the Systray widget.

What happens if you add:
``` python
screens = [Screen()]
```

**Reply by antidoid** [reply]
Voila, and it works now without even hiding the bar, because now I assume there is no bar. That's great. Thank you for your help. I was stuck on this from long time.

Out of curiosity, How does that line help?

**Reply by elParaguayo** [reply]
You need to define screens in your config in the `screens` variable. It's a required section of the config. As you removed it entirely, qtile uses the default value for `screens` which is this:
https://github.com/qtile/qtile/blob/6620786f801f56fc61716514139044108eaef743/libqtile/resources/default_config.py#L125-L152

By adding the line from my previous comment, you've now defined `screens` to be a `Screen` object with no bars and, as `screens` is now included in your config, the default value is not used.

**Reply by antidoid** [reply]
Got it. Thanks for answering and your time.
